cloud/microsoft/office365/exchange/mode/mailboxusage.pm

```diff
@@ -245,25 +249,31 @@ sub manage_selection {
     $self->{global} = { storage_used_active => 0, storage_used_inactive => 0 };
     $self->{mailboxes} = {};
 
-    my $results = $options{custom}->office_get_exchange_mailbox_usage();
+    my $results = $options{custom}->office_get_exchange_mailbox_usage(param => "period='D7'");
+    my $results_daily = [];
+    if (scalar(@{$results})) {
+       $self->{active}->{report_date} = @{$results}[0]->{'Report Refresh Date'};
+       #$results_daily = $options{custom}->office_get_exchange_mailbox_usage(param => "date=" . $self->{active}->{report_date});
```

Comment:
Why not here?

Reply:
Because the [`getMailboxUsageDetail` API call](https://docs.microsoft.com/en-us/graph/api/reportroot-getmailboxusagedetail?view=graph-rest-1.0) only support _period_ request, not _date_ request. It's the only one like this.